### DISCUSSION TITLE
QUIC compatibility

### DISCUSSION THREAD

**Discussion by radiumatic** [XTLS/Xray-core#3980] [Q&A]
Since QUIC seems to have been removed and splithttp with h3 ALPN is the replacement (I don't understand how they're even similar), how is the compatibility with previous versions and other proxy platforms? Is vless-over-splithttp3 a drop-in replacement for vless-over-QUIC?

**Comment by lolitakon** [comment]
Yes, splithttp is more compatible than the original quic in terms of concealment and high concurrency, but it also introduces more delays, and there are still some compatibility issues. I think it is necessary to restore the original quic flow.

**Reply by radiumatic** [reply]
I think you misunderstood. If I have a QUIC server (xray-core, old version or sing-box, even clash), is it usable with splithttp? (Without modification to the server, obviously)

**Reply by lolitakon** [reply]
nope，splithttp h3 is not the original quic stream，and the splithttp of the new and old versions of xray core are also incompatible

**Reply by radiumatic** [reply]
That's really stupid. Why was it removed? To save some KBs of binary size?
Sing-Box and Clash are apparently opposed to the idea of splithttp. This is going to create even more problems.
